service/src/main/resources/HTMLTemplate.html

```diff
@@ -1440,9 +1447,10 @@ <h3 class="heading heading--two">Applicant</h3>
     <div class="data-list__row">
       <dt class="data-list__title">Telephone number</dt>
       <dd class="data-list__definition">
-        {% if case_data.applicant is not empty and
-        case_data.applicant.telephone is not empty %}
-          {{ case_data.applicant.telephone }}
+        {% if applicant.value.party.telephoneNumber is not empty %}
+        {% if applicant.value.party.telephoneNumber.telephoneNumber is not empty %}
```

Comment:
Second condition (`{% if applicant.value.party.telephoneNumber.telephoneNumber is not empty %}`) should not be on its own, it should be joined with previous one.

In other words it should be `{% if applicant.value.party.telephoneNumber is not empty and applicant.value.party.telephoneNumber.telephoneNumber is not empty %}`

Current implementation will result in displaying blank cell instead of `-` in case ` applicant.value.party.telephoneNumber` is populated but ` applicant.value.party.telephoneNumber.telephoneNumber` is null.

The same should be corrected in other places.

Reply:
👍